### ISSUE TITLE
qsyncthingtray fail to build

### ISSUE THREAD

**Issue by sib-null** [NixOS/nixpkgs#42802]
## Issue description
`qsyncthingtray` failed to build when I did a `nixos-rebuild switch --upgrade`

```
CMake Error at CMakeLists.txt:131 (qt5_use_modules):
  Unknown CMake command "qt5_use_modules".


-- Configuring incomplete, errors occurred!
See also "/tmp/nix-build-qsyncthingtray-0.5.8.drv-0/source/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/1fywy7y216zjv7vkv9a7hc4rj8865bcp-qsyncthingtray-0.5.8.drv' failed with exit code 1
```

Possibly related to #42352 and #42351

### Steps to reproduce
Install or upgrade `qsyncthingtray`

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.16.11, NixOS, 18.09pre140958.696c6bed4e8 (Jellyfish)`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.0.2`
 - channels(root): `"nixos-18.09pre144567.e686bd27719"`
 - channels(bill): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

**Comment by jtojnar** [comment]
Yep, the same issue as #42352

**Comment by orivej-nixos** [comment]
Fixed by #42351
